src/surrogates/inc/InterpolationSurrogateBuilder.h

```diff
@@ -47,7 +47,7 @@ namespace QUESO
     //! Constructor
     /*! We do not take a const& to the data because we want to compute and
         set the values directly. */
-    InterpolationSurrogateBuilder( InterpolationSurrogateData<V,M>& data );
+    InterpolationSurrogateBuilder( InterpolationSurrogateDataSet<V,M>& data );
```

Comment:
No, I think what you have is fine.  Filling a vector with one element isn't a big deal and I imagine most use-cases involve several quantities to calibrate rather than just one.